fusion-cli/build/get-webpack-config.js

```diff
@@ -132,9 +132,14 @@ function getWebpackConfig(opts /*: WebpackConfigOpts */) {
     legacyPkgConfig = {},
     worker,
   } = opts;
-  const main = 'src/main.js';
+  const main = fusionConfig.main || 'src/main.js';
```

Comment:
Could we instead just check for the existence of `src/main.js` or `src/main.ts` and update our webpack accordingly?